lib/page_ez/visitors/registered_name_visitor.rb

```diff
@@ -32,7 +32,7 @@ def track_method_added(name, construction_strategy)
 
         found = @declared_constructors[parent_id].find { _1 == [name, construction_strategy] }
 
-        if found && !found[1].is_a?(PageEz::MethodGenerators::HasOneDynamicSelector)
+        if found && found[1]&.selector_type != :dynamic
```

Comment:
I'm curious why you chose to use `!= :dynamic` vs `== :static`? Is the intent "anything that's not dynamic should raise" vs "only raise for static selectors"?

Reply:
@eebs you're spot on re: anything that's not dynamic should raise